src/slices/directanswer.ts

```diff
@@ -15,6 +15,10 @@ const reducers = {
 
 /**
  * Registers with Redux the slice of {@link State} pertaining to direct answers.
+ *
+ * @param prefix - The prefix to access the part of the Redux store specific to the
+ *                 AnswersHeadless instance
+ * @returns The {@link Slice} for direct answers
```

Comment:
is it possible to have these slice creators have some kind of shared type? Then maybe we can consolidate this documentation instead of having it copied for every slice creator we have

Reply:
ended up removing these because the slice creators aren't exported to API extractor